src/InstrumentationEngine.Api/InstrumentationEngine_api.cpp

```diff
@@ -9,8 +9,8 @@
 /* at Mon Jan 18 19:14:07 2038
  */
 /* Compiler settings for InstrumentationEngine.idl:
-    Oicf, W1, Zp8, env=Win64 (32b run), target_arch=AMD64 8.01.0622 
-    protocol : all , ms_ext, c_ext, robust
+    Oicf, W1, Zp8, env=Win32 (32b run), target_arch=X86 8.01.0622 
```

Comment:
I don't think it's necessary, but I'll revert these comments.

Reply:
These get changed based on change in compilation. I'm not sure if there's a way to prevent it but it's benign